lib/index.js

```diff
@@ -17,11 +17,12 @@
 ************************************************************************ */
 
 require("qooxdoo");
-if (qx.tool === undefined) 
-  qx.tool = {};
+if (qx.tool === undefined) {
+  qx.tool = {}; 
+}
 require("./qx/tool/cli.js");
 require("./qx/tool/compiler.js");
 
-debugger;
```

Comment:
this is really useful (almost essential)

Reply:
no Problem...